src/field/pokemon.ts

```diff
@@ -3451,6 +3451,13 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
     return this.summonData.tags.find((t) => tagTypes.includes(t.tagType)) as T | undefined;
   }
 
+  public hasTag(...tagTypes: BattlerTagType[]): boolean {
```

Comment:
Public function should have documentation.

```suggestion

  /**
   * Helper function to check if a Pokemon has any of the input tag types
   * @param tagTypes the battler tag types to search for
   * returns true if the Pokemon has at least one of the battler tags, false otherwise
   */
  public hasTag(...tagTypes: BattlerTagType[]): boolean {
```